tests/gui/models/supported-formats.spec.js

```diff
@@ -123,6 +123,35 @@ describe('Browser: SupportedFormats', function() {
 
     });
 
+    describe('.looksLikeWindowsImage()', function() {
+
+      it('should return true if filename is `en_windows_10_multiple_editions_version_1607_updated_jan_2017_x64_dvd_9714399.iso`',
```

Comment:
Since all these tests are very similar, we could have an array of paths, and loop through them creating test cases dynamically. For example:

```js
_.each([
  '/path/to/Win10_1607_SingleLang_English_x32.iso',
  '/path/to/Win10_1607_SingleLang_English_x32.iso',
  '/path/to/Win10_1607_SingleLang_English_x32.iso',
  '/path/to/Win10_1607_SingleLang_English_x32.iso'
], (imagePath) => {

  it(`should return true if filename is ${imagePath}`, function() {
    const looksLikeWindowsImage = SupportedFormatsModel.looksLikeWindowsImage(imagePath);
    m.chai.expect(looksLikeWindowsImage).to.be.true;
  });

});
```

We should also make sure to try this with Windows style paths.